testjson.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
What is the difference between this one and the NOtestjson.json?

Reply:
This is the 'real' test configuration file. We will soon feature PROFILE_NAME.json files for differing profiles.